syft/formats/common/spdxhelpers/to_format_model.go

```diff
@@ -511,6 +513,28 @@ func toFileTypes(metadata *source.FileMetadata) (ty []string) {
 	return ty
 }
 
+func toOtherLicenses(catalog *pkg.Catalog) []*v2_3.OtherLicense {
```

Comment:
This should return `[]*spdx.OtherLicense` and we shouldn't import `v2_3` -- everything in the `spdx` package is an alias to the latest version (in this case, v2_3)

Reply:
I was wondering about that. OK, I can get that.

Reply:
Done